x/metadata/client/cli/tx.go

```diff
@@ -1033,6 +1076,46 @@ func RemoveContractSpecificationCmd() *cobra.Command {
 	return cmd
 }
 
+// RemoveContractSpecFromScopeSpecCmd removes a contract spec from scope spec command
+func RemoveContractSpecFromScopeSpecCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "remove-contract-spec-to-scope-spec contract-specification-id scope-specification-id",
+		Short:   "Remove an existing contract specification from a scope specification",
+		Aliases: []string{"rcsfss"},
+		Args:    cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			contractSpecID, err := types.MetadataAddressFromBech32(args[0])
+			if err != nil {
```

Comment:
```suggestion
			if err != nil  || !contractSpecID.IsContractSpecificationAddress() {
```
verify address type... note: will need to handle/create an error for this to use with following line